lib/src/contracts/uhst_event_handlers.dart

```diff
@@ -3,7 +3,7 @@ part of uhst_contracts;
 typedef OpenHandler = void Function();
 typedef MessageHandler = void Function({required String message});
 typedef ExceptionHandler = void Function({required dynamic exception});
-typedef CloseHandler = void Function();
+typedef CloseHandler = void Function({required String hostId});
```

Comment:
hostId here is not useful

Reply:
the reason is that as onClose will be emitted to event stream, and in the moment when listener will be called there will be no hostId in socket as it will be closed - and it will be null

Reply:
can you think of a good use case (maybe from your experience working on the SCRUM Poker App) where the hostId will be useful in onClose? I am probably missing something.

Reply:
No, I don't find such cases in SCRUM Poker App)
Fixed